drivers/flash/spi_nor.c

```diff
@@ -314,6 +391,7 @@ static int spi_nor_configure(struct device *dev)
 	const struct spi_nor_config *params = dev->config->config_info;
 
 	data->spi = device_get_binding(DT_INST_0_JEDEC_SPI_NOR_BUS_NAME);
+	printk("Dev %s at %p\n", DT_INST_0_JEDEC_SPI_NOR_BUS_NAME, data->spi);
```

Comment:
Use logging infrastructure instead of printk.

Reply:
Left-over debug message, will remove.